### PR TITLE
BETA: Register zemerik.is-a.dev

### PR DESCRIPTION
**Pull request by Zemerik**
Added `zemerik.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/zemerik.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Zemerik",
+        "email": "zemerikY@gmail.com"
+    },
+    "record": {
+        "CNAME": "zemerik.github.io"
```
Comment (DEV-DIBSTER) [inline domains/zemerik.json:7]: 404 CNAME record.

**Review by DEV-DIBSTER — DISMISSED** [review]

**Comment by Zemerik** [comment]
@DEV-DIBSTER Fixed👍

**Review by priyazsh — APPROVED** [review]

**Comment by Zemerik** [comment]
@DEV-DIBSTER update??

**Review by wdhdev — APPROVED** [review]

**Comment by Zemerik** [comment]
Thank you

**Pull request merged**